ios-task/README.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+This task is intended for people who interview for our iOS Software Engineer roles.
+
+Please make sure you read [our expectations](../README.md#what-we-are-looking-for) first before starting the following tasks.
+Take your time to polish your project. 
+Testing is very important for us, add tests where is possible and design the app in a way that should be easy to be tested.
+
+## iOS News App
+
+Build an iOS application that shows the latest news from the United States, using the [newsapi.org](https://newsapi.org) service.
+
+Running the app we should be able to see a list with the latest **US top headlines news** (see [newsapi.org documentation](https://newsapi.org/docs/)).
+Each news item in the list should have the description of the article, the author, and the image associated to it. 
+If the news doesn't have an image, present an image placeholder of your choice.
+
+If the user clicks on a news item it will navigate to a page displaying the full news article selected.
```

Comment:
This seems fairly open-ended - what's the "happy path" we expect candidates to take here? Building out another view for "article" page? Rendering a webview? Launching browser?

Reply:
given the article is an URL and we must also present somehow the number of comments and number of likes in the same page, it should be a view with two labels and a webview.
But got your point, I'll explain it better

Reply:
done